src/viam/app/app_client.py

```diff
@@ -1781,13 +1852,48 @@ async def delete_fragment(self, fragment_id: str) -> None:
         request = DeleteFragmentRequest(id=fragment_id)
         await self._app_client.DeleteFragment(request, metadata=self._metadata)
 
+    async def get_fragment_history(self,
+                                   id: str,
+                                   page_token: Optional[str] = "",
+                                   page_limit: Optional[int] = 10) -> list[FragmentHistoryEntry]:
+        """Get fragment history.
+
+        ::
+
+            await cloud.get_fragment_history(
```

Comment:
we should assign some value to this (`fragment_history = await cloud.get_fragment_history(...)`).